grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_form.ts

```diff
@@ -24,9 +26,25 @@ export class OsqueryForm extends FlowArgumentForm<OsqueryArgs> implements
     query: new FormControl(this.defaultQueryDisplayed, Validators.required),
     timeoutMillis: new FormControl(null, Validators.required),
     ignoreStderrErrors: new FormControl(null),
+    fileCollectColumns: new FormControl([]),
   });
 
-  settingsShown = false;
+  fileCollectionSettingsShown = false;
+  lowLevelSettingsShown = false;
+
+  readonly separatorKeysCodes: number[] = [ENTER, COMMA];
+
+  get fileCollectColumnsControl() {
+    const control = this.form.get('fileCollectColumns');
+    if (!control) {
```

Comment:
Please use https://github.com/google/grr/blob/252af5d85f02121a75f8d83ebbee8e3d7e4f5fc9/grr/server/grr_response_server/gui/ui/lib/preconditions.ts#L17 for this check.